manifest.toml

```diff
@@ -61,6 +61,12 @@ ram.runtime = "512M"
     choices = ["yes", "no"]
     default = "yes"
 
+    [install.enable_notify_push]
+    ask.en = "Configure the High Performance Backend?"
+    ask.fr = "Configurer le Backend Hautes Performances ?"
+    type = "boolean"
+    default = false
```

Comment:
Should we add a scenario in tests.yml where this is set to true ?

Reply:
Yes we should, but I still work to do on this PR.

For example, this: (`expand Alternatively, you can do this automatically via systemctl by creating the following systemd service and path (click to expand)`)

https://github.com/nextcloud/notify_push?tab=readme-ov-file#starting-the-service